amethyst_core/src/transform/components/local_transform.rs

```diff
@@ -64,16 +59,36 @@ impl LocalTransform {
         self
     }
 
-    /// Move relatively to its current position and orientation.
+    /// Move forward relative to current position and orientation.
+    pub fn move_forward(&mut self, orientation: &Orientation, amount: f32) -> &mut Self {
+        self.move_local(orientation.forward.into(), amount)
+    }
+
+    /// Move backward relative to current position and orientation.
+    pub fn move_backward(&mut self, orientation: &Orientation, amount: f32) -> &mut Self {
+        self.move_local((-orientation.forward).into(), amount)
+    }
+
+    /// Move right relative to current position and orientation.
     pub fn move_right(&mut self, orientation: &Orientation, amount: f32) -> &mut Self {
         self.move_local(orientation.right.into(), amount)
     }
 
-    /// Move relatively to its current position and orientation.
+    /// Move left relative to current position and orientation.
+    pub fn move_left(&mut self, orientation: &Orientation, amount: f32) -> &mut Self {
+        self.move_local((-orientation.right).into(), amount)
+    }
+
+    /// Move up relative to current position and orientation.
     pub fn move_up(&mut self, orientation: &Orientation, amount: f32) -> &mut Self {
         self.move_local(orientation.up.into(), amount)
     }
 
+    /// Move down relative to current position and orientation.
+    pub fn move_down(&mut self, orientation: &Orientation, amount: f32) -> &mut Self {
+        self.move_up( orientation, -amount)
```

Comment:
Why does down is implemented via up while other pairs are implemented independently?